### PR TITLE
Adding 3D scanning video links with docs

### PR DESCRIPTION
**Pull request by alextha-unity**
Adding video links that @SCRktkt created.  Credits also to Sharmila for helping to get these posted to YouTube.
These will ultimate reside on our Perception Hub page but we will maintain links on SynthDet for now.

### CODE REVIEW

File: docs/CreatingAssets.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Creating your own assets
+<img src="images/CerealBox.png" align="middle"/>
+
+
+Using the SynthDet example project, you are able to create a custom dataset for your own objects.
+Creating 3D content for synthetic data and bringing into a Unity project can be challenging.  We have provide you some tutorial videos to help with the scanning and import process.  The videos below are primarily focused on creating content for object detection ML models.  The same approaches can be used for creating digital twins of other objects, but for other computer vision tasks, there are other strategies that should be employed, including making use of existing assets on the [Unity Asset Store](https://assetstore.unity.com/) or creating assets by hand.  For more information, please [reach out to us](mailto:perception@unity3d.com) to see how we can help!
```
Comment (JonathanHUnity) [inline docs/CreatingAssets.md:6]:
```suggestion
Creating 3D content for synthetic data and bringing into a Unity project can be challenging.  We have provided you some tutorial videos to help with the scanning and import process.  The videos below are primarily focused on creating content for object detection ML models.  The same approaches can be used for creating digital twins of other objects. For other computer vision tasks there are other strategies that may be employed, including making use of existing assets on the [Unity Asset Store](https://assetstore.unity.com/) or creating assets by hand.  For more information, please [reach out to us](mailto:perception@unity3d.com) to see how we can help!
```

Comment (JonathanHUnity) [reply docs/CreatingAssets.md:6]: Just some grammar and flow suggestions.

**Review by JonathanHUnity — CHANGES_REQUESTED** [review]
Great. Some minor fixes and suggestions here for you.

**Review by alextha-unity — COMMENTED** [review]
Updated per feedback

**Review by JonathanHUnity — APPROVED** [review]

**Pull request merged**